bats/fb-test-foreman.bats

```diff
@@ -10,7 +10,7 @@ set -o pipefail
 }
 
 @test "check web app is up" {
-  curl -sk "https://localhost$URL_PREFIX/users/login" | grep login-form
+  [ `curl -sk "https://localhost$URL_PREFIX/status" | jq .status` == '"ok"' ] 
```

Comment:
We don't have jq installed. I also don't think it's present in downstream where we don't have EPEL. That means this will still fail.

Reply:
maybe we can use something like: `curl -sk "http://localhost:5000/users/login" | grep -E 'login-form|login-page'`

Reply:
https://github.com/theforeman/forklift/pull/970